xmake/actions/config/scangen.lua

```diff
@@ -156,5 +156,5 @@ function main()
     end
 
     -- trace
-    cprint("${bright}xmake.lua generated, scan ok!${clear}")
+    cprint("${color.success}xmake.lua generated, scan ok!${clear}")
```

Comment:
we need not add `${clear}` at tail, because cprint will automatically add it internally. 

https://github.com/xmake-io/xmake/blob/4a7f397b9036a3c31cad090732d758794555e40c/xmake/core/base/colors.lua#L326

we need only add `${color.success}`.

```lua
    cprint("${color.success}xmake.lua generated, scan ok!")
```

Reply:
Sure, I'll look through the code base now.

Reply:
Done ... added to the PR.

I removed all the `${clear}` elements which were the last portion of a `cprint(...)`.